src/rngs/os.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Developers of the Rand project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Interface to the random number generator of the operating system.
+
+use getrandom::getrandom;
+use rand_core::{CryptoRng, RngCore, Error, ErrorKind, impls};
+
+/// A random number generator that retrieves randomness from from the
+/// operating system.
+///
+/// This is a zero-sized struct. It can be freely constructed with `OsRng`.
+/// 
+/// The implementation is provided by the [getrandom] crate. Refer to
+/// [getrandom] documentation for details.
+///
+/// [getrandom]: https://crates.io/crates/getrandom
```

Comment:
I think it will be more convenient to provide link to docs.rs instead.

Reply:
I disagree. The crates page tells me immediately (1) what the purpose of the crate is, (2) what the current version is, and (3) links the docs.rs page and (4) links the repo. Docs.rs doesn't seem to have any outward-going links and starts with the API documentation, not the README.

Reply:
crates.io link is good for README, but I don't think it's nice to jump between services when you read documentation. If you will look carefully docs.rs page has everything you listed as well. Also I think we should update root docs to be closer to readme (e.g. add usage example to the beginning).

>doesn't seem to have any outward-going links

Hover your cursor over `getrandom-0.1.0` on docs.rs page.